lambda/src/server/slack.cljs

```diff
@@ -44,7 +44,7 @@
                             :headers {:Authorization (str "Bearer " bot-token)}
                             :body (clj->js body) #_(.stringify js/JSON (clj->js body))})))
 
-(tasks/alias! ::post post+)
+(tasks/register-var! `post+)
```

Comment:
I don't _think_ we can avoid pre-registering these fns, because of renaming during advanced-compile